.github/workflows/validate.yaml

```diff
@@ -46,11 +46,11 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Build Connections
-      run: CC=clang CXX=clang++ bazel build --copt='-DGITHUB_BUILD' //connections:core
+      run: CC=clang-15 CXX=clang-15++ bazel build --copt='-DGITHUB_BUILD' //connections:core
     - name: Build Presence
-      run: CC=clang CXX=clang++ bazel build --copt='-DGITHUB_BUILD' //presence
+      run: CC=clang-15 CXX=clang-15++ bazel build --copt='-DGITHUB_BUILD' //presence
```

Comment:
is there a reason for clang 15 specifically?

Reply:
I think clang-14 is missing some of the C++ 20 standard features. The build was failing on clang-14 when I updated the standard, but luckily clang-15 also comes pre-installed on the machines

Reply:
Got it, makes sense. Should we have a tracking bug to monitor when clang 15 becomes the default and remove this explicit version then?

Reply:
Personally, I have no problems with leaving it as the explicit version, rather than using whatever the build machine happens to default too, but we can open a tracker to change it in the future if you prefer that as well